website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -48,15 +48,20 @@ Please read through the entire procedure before beginning the upgrade. There are
 
     ```bash
     export KARPENTER_NAMESPACE=kube-system
-    export KARPENTER_VERSION=1.0.0
+    export KARPENTER_IAM_ROLE_ARN="arn:${AWS_PARTITION}:iam::${AWS_ACCOUNT_ID}:role/${CLUSTER_NAME}-karpenter"
     export AWS_PARTITION="aws" # if you are not using standard partitions, you may need to configure to aws-cn / aws-us-gov
     export CLUSTER_NAME="${USER}-karpenter-demo"
     export AWS_REGION="us-west-2"
     export AWS_ACCOUNT_ID="$(aws sts get-caller-identity --query Account --output text)"
     ```
 
-4. Update your existing policy using the following:
-   Notable Changes to the IAM Policy include `SSM:GetParameter` to `SSM:GetParametersByPath` and additional tag-scoping for the `eks:eks-cluster-name` tag for instances and instance profiles.
+4. Set environment variables for first upgrading to v0.37.1
```

Comment:
This should probably be a function of whatever minor version they're on, right? We're not requiring users migrate to v0.37 from v0.33.

Reply:
Yeah, you are right on that one. I will update